data/campaigns/tutorial/scenarios/02_Tutorial_part_2.cfg

```
@@ -323,6 +323,20 @@
 
 A full list of abilities and weapons specials, along with traits, may be found in help."
         [/message]
+
+        # Triggers when a second Shaman is recruited
+        [event]
+            name=recruit
+            [filter]
+                side=1
+                type=Elvish Shaman
+            [/filter]
+
+            [message]
+                speaker=Galdrad
+                message= _ "Having more than one shaman will mean that you can heal more units, and in separate areas of the battlefield. However, a unit next to two or more shamans will still only heal 4 hitpoints per turn."
```

Mention Druid's 8HP healing ability? (even though the player probably can't level a Shaman in this scenario)

---

I'd prefer to keep the message shorter, and explain the Druid's ability when a Shaman levels up into a Druid.

---

Maybe this, but I'm not too happy with it:

```suggestion
                message= _ "Having more than one shaman will mean that you can heal more units, and in separate areas of the battlefield. However, a unit next to two or more shamans will still only heal 4 hitpoints per turn."
             [/message]
             [message]
                speaker=Galdrad
                message= _ "In campaigns, shamans can be advanced to Elvish Druids, who have a more powerful healing ability. For now, however, concentrate on keeping your shamans alive so they can continue to heal your other troops." 
```

---

Thanks, the suggestions have improved the PR.

I now have 2 lines said by the healers, and 1 from Galdrad.